ember-amount-input/src/components/amount-input.ts

```diff
@@ -5,6 +5,7 @@ import './amount-input.css';
 const KEY_CODE_E = 69;
 const KEY_CODE_FULLSTOP = 190;
 const KEY_CODE_COMMA = 188;
+const KEY_CODE_MINUS = '-';
```

Comment:
nitpick: going by the browser API naming it's not a key code
```suggestion
const KEY_MINUS = '-';
```